### PR TITLE
[tw] Localize `eBPF` for Traditional Chinese

### LINKED ISSUE #2543: [tw] Localize `eBPF` for Traditional Chinese
- Localize `eBPF` in Traditional Chinese
- https://glossary.cncf.io/ebpf/

### PR DESCRIPTION
**Pull request by tico88612**
### Describe your changes

Origin content from https://glossary.cncf.io/ebpf

### Related issue number or link (ex: `resolves #issue-number`)

resolves #2543 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Review by johnlinp — APPROVED** [review]
LGTM

File: content/zh-tw/ebpf.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+在大多數情況下，這種方法都能正常執行。不過，在某些情況下，開發人員需要更靈活的低階系統存取方式。
+可觀察性、安全性和網路功能就是很好的範例。
+為此，我們可以使用 Linux 核心模組，在不修改核心原始碼的情況下延伸核心基礎。
+雖然使用 Linux 核心模組有很多好處，但也會帶來安全風險。
+由於 Linux 核心模組在核心空間內執行，它們可能會導致核心崩潰，而核心一旦崩潰，整個機器也會當機。
+此外，核心模組擁有更高的權限，可以直接存取系統資源。如果沒有適當的安全保護，攻擊者就會利用這些漏洞。
+
+## 如何幫助我們 {#how-it-helps}
+
+與 Linux 核心模組相比，eBPF 為執行使用者定義的程式提供了一個更可控、更封閉的環境。
```
Comment (tico88612) [inline content/zh-tw/ebpf.md:32]: > 原文：eBPF provides a more controlled and contained environment for executing user-defined programs than Linux Kernel modules. 

more controlled and contained 翻成「更可控、更封閉」，有點怪怪的，但我現在想不到有沒有更好的寫法。

Comment (hwchiu) [reply content/zh-tw/ebpf.md:32]: 相對於純 kernel module，透過 eBPF 的確會讓整個程式更容易被執行與控制，所以我覺得更可控沒有太大問題。

至於「更封閉」我覺得稍微怪怪的，畢竟比較起來不一定是比較封閉

[contained](https://www.dictionary.com/browse/contained)
showing restraint or calmness; controlled; poised:

以形容詞來看似乎也有被控制的意思，還是乾脆就寫更可控就好XD

Comment (johnlinp) [reply content/zh-tw/ebpf.md:32]: image

也可以考慮翻譯成 "更受限制"

Comment (tico88612) [reply content/zh-tw/ebpf.md:32]: contained 確實是有控制意思，所以應該就不用兩個詞語了。不過 ChatGPT 修正還不錯，預計會使用：

> eBPF 提供比 Linux 核心模組更可控的環境，以執行使用者自訂的程式。

**Review by hwchiu — APPROVED** [review]
LGTM

**Pull request merged**